Client/core/Graphics/CPrimitiveBatcher.cpp

```diff
@@ -100,24 +100,10 @@ void CPrimitiveBatcher::UpdateMatrices(float fViewportSizeX, float fViewportSize
 void CPrimitiveBatcher::SetDeviceStates()
 {
     // Set states
-    m_pDevice->SetRenderState(D3DRS_ZENABLE, D3DZB_FALSE);
-    m_pDevice->SetRenderState(D3DRS_ZFUNC, D3DCMP_LESSEQUAL);
-    m_pDevice->SetRenderState(D3DRS_ZWRITEENABLE, FALSE);
-    m_pDevice->SetRenderState(D3DRS_CULLMODE, D3DCULL_NONE);
```

Comment:
Why do you remove stuff not related to blending? Deleting this one will result in out of order primitives not rendering because of backface culling

Reply:
Oops

Reply:
@forkerer Those states are already correctly set in `CGraphics::SetBlendModeRenderStates` https://github.com/multitheftauto/mtasa-blue/blob/4c3bbb91468205d8e55c53e6a2cfe96ca306f2c7/Client/core/Graphics/CGraphics.cpp#L344

Reply:
That's interesting, i remember i had some issues with culling when writing it, that's why it was set explicitly there. Maybe I'm misremembering something, sorry for false alarm